it/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AddressLookupFrontendConnectorSpec.scala

```diff
@@ -13,23 +13,21 @@ class AddressLookupFrontendConnectorSpec extends BaseISpec with MetricTestSuppor
 
   "getAddressDetails" should {
     "convert the JSON returned by address-lookup-frontend into an object" in {
-      givenCleanMetricRegistry()
-
-      val addressId = "id"
-      val addressLine1 = "10 Other Place"
-      val addressLine2 = "Some District"
-      val addressLine3 = "Line 3"
-      val town = "Our town"
-      val postcode = "AA1 1AA"
-      givenAddressLookupReturnsAddress(addressId, addressLine1, addressLine2, addressLine3, town, postcode)
-      val connector = app.injector.instanceOf[AddressLookupFrontendConnector]
-      val address = await(connector.getAddressDetails(addressId))
-      address shouldBe AddressLookupFrontendAddress(
-        lines = Seq(addressLine1, addressLine2, addressLine3, town),
-        postcode = Some(postcode),
-        country = Country("GB", Some("United Kingdom")))
-
-      timerShouldExistAndBeUpdated("ConsumedAPI-Address-Lookup-Frontend-getAddressDetails-GET")
+      withMetricsTimerUpdate("ConsumedAPI-Address-Lookup-Frontend-getAddressDetails-GET") {
+        val addressId = "id"
+        val addressLine1 = "10 Other Place"
+        val addressLine2 = "Some District"
+        val addressLine3 = "Line 3"
+        val town = "Our town"
+        val postcode = "AA1 1AA"
```

Comment:
in code, we should use normalised style of data. so all inputs should be evaluated after form submission, otherwise there is no spacing.
if you're using this in tests, then modify the postcode in that particular test, but I see this is supposed to be a valid postcode so keep it 100% valid

Reply:
It's all good. 
The DES compliant validation is performed upon returning from the address lookup in the `SubscriptionController.returnFromAddressLookup` endpoint which is covered in the `SubscriptionControllerISpec` tests.